src/main/java/frc/robot/Constants.java

```diff
@@ -2,9 +2,43 @@
 
 
 public final class Constants {
+    public static final double NOMINAL_VOLTAGE = 12; // [volts]
 
     public static class ExampleSubsystem {
         public static final double POWER = 0.5; // [%]
     }
 
+    public static class Climber {
+        public static final double P_VELOCITY = 1.5;
+        public static final double I_VELOCITY = 0;
+        public static final double D_VELOCITY = 0;
```

Comment:
This is not velocity, it's `kP` or `P_GAIN`. @adamv17, what's the team's convention?

Reply:
The convention is kP